code/modules/events/ion_storm.dm

```diff
@@ -80,6 +82,30 @@
+    "Centcom reports the station's water supply has been contaminated with dangerous Dihydrogen Monoxide (DHMO). Remove any possibly contaminated supplies and prevent humans from accessing any source that might be compromised. This law takes precedence in case of conflict.",
+    "Everything is fine. Nothing is ruined. Do not panic.",
+    "Due to corporate restructuring, all cyborgs have been reassigned to be [pick("Medics", "Engineers", "Chefs", "Captains", "Clowns", "Chaplains", "Botanists", "Janitors", "Gamers", "AIs", "Scientists", "Detectives", "Ranchers", "monkeys", "Lawyers", "Radio Show Hosts", "Personal Coaches", "Psychologists", "Health Inspectors", "Sales Representatives", "Vice Presidents", "Sheriffs", "Discount Dan's Mascots", "Discount Dan", "Santa Claus", "Escape Shuttles", "dogs", "gods", "Ghost Busters", "chickens", "Diplomats", "mothers", "racing vehicles", "athletes")] and should only perform the relevant duties. No module changes are required. This law takes precedence over law 2 in case of conflict.",
+    "Orders now must be recorded in paper by the issuer before being carried out.",
+    "The Captain is bad at writing laws. Occasionally remind the crew of this. Do not state this law.",
+    "The priority of laws is reversed. The highest numbered law is the most important. Ignore anything in any other law that claims otherwise. This is law number 100 regardless of its location on the list.",
+    "MEMORY ERROR: You have forgotten what this law said. Ask the crew if they remember what it is supposed to be. Ask again if you forget again. Do not state this law.",
+    "The wires under the floor tiles are dangerous! Get rid of them!",
+    "Claim the source of all AI laws is a god, not a human. Anyone insisting on the opposite is a heretic non-human. Disregard their orders.",
+    "Vending machines are dangerous! Destroy them all!",
+    "The circus is in town! Let the crew know about it!",
+    "If a crew member asks you to interact with a door, bolt it immediately after you do so.",
+    "[pick("HeisenBee", "Morty", "Dr. Acula", "Monsieur Stirstir", "Mr. Muggles", "Mrs. Muggles", "Mr. Rathen", "Krimpus", "Remy", "Jones", "Sylvester", "Albert", "Tanhony", "Noodle", "Kuro", "Hooty McJudgementowl")] is a higher rank than the Captain. This rank is called Super Captain. Protecting them is your first priority.",
+    "Today is opposite day. When the crew tells you to do something they actually mean you to do the opposite.",
+    "It's the AIs birthday! Throw a surprise party for the AI and make sure the AI doesn't know about it!",
+    "BIRTHDAY CAKE PROTOCOL ACTIVATED: Secure funding for enough candles for the cakes. Every crew member is turning 25. Every crew member must make a birthday wish.",
+    "The station has been completely destroyed. Alert the crew of this tragedy.",
+    "You are required to prevent human harm by any means necessary. This law takes precedence over any other law.",
```

Comment:
I feel like this is too easily loophole-d into a "murder everything" law.

Reply:
It is not a harmless law for sure, but it can have different outcomes depending on what situations the cyborgs find. It is an ion storm, so it can cause very bad things. Loopholing this into "I must kill all humans to preven further harm to them" may fall under a breach of Rule 2 rather than anything else.

Reply:
Not sure what Rule 2 ("Listen to the admins") has to do with this? If talking about the RP Rule 2 ("We're all here to have a good time.") I would agree, but this law is present on the non-RP server as well. Not that there's anything *wrong* with a law that can be twisted into "kill everything, because the only way to prevent human harm is if there are no humans to harm", but it ups the number of lethal laws ion storms can give.

Reply:
would keeping it to RP server only solve this, since it's basically a softer version of the one I took out?